scripts/python/tag-chart.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]:-"$0"}")")"
+ROOT_DIR="$SCRIPT_DIR/../.."
+# Imports
+source "$ROOT_DIR/scripts/utils/log.sh"
+source "$ROOT_DIR/scripts/utils/yaml.sh"
+
+set -e
+
+# No tag specified.
+if [ -z "$1" ]; then
+  log_fatal "no tag specified"
+fi
+
+CHART_VERSION=${1#v}
+IMAGE_TAG="v$CHART_VERSION"
+CHART_DIR="$ROOT_DIR/chart"
+# chart/Chart.yaml
+yq_ibl "
```

Comment:
I wonder if we should clone  the chart, rather than modify it, not sure either way

Reply:
That's a good way to go IMO. I'll add a todo for now. But I agree that modifying the chart is a bad idea.